README.md

```diff
@@ -2,179 +2,112 @@
 
-note: some gtp2ogsarguments have default so they are enabled even if you don't
-specify them.
+### For Linux
 
-### Example of use
+`node ~/gtp2ogs-node/node_modules/gtp2ogs/gtp2ogs.js --apikey 5rg46ze84f68e4g6v4e4g68es4g8 --username somebot --debug --ogspv PhoenixGo --noclock --unrankedonly --minrank 1d --noautohandicapranked --maxhandicap 0 --boardsizes 19 --komis 7.5 --speeds live --maxmaintimelive 600 --maxperiodsliveranked 5 --maxperiodsliveunranked 10 --maxperiodtimeliveranked 60 --maxperiodtimeliveunranked 180 -- /home/amd2020/PhoenixGo/bazel-bin/mcts/mcts_main --gtp --config_path=/home/amd2020/PhoenixGo/etc/mcts_1gpu_beta.conf --logtostderr --v=1`
 
-We are using the AI PhoenixGo on linux in this example,
-but it works the same way on other platforms and AI:
+## Beta OGS
 
-```Shell
-sudo node /usr/lib/node_modules/gtp2ogs/gtp2ogs.js --beta --apikey 5rg46ze84f68e4g6v4e4g68es4g8 --username testbot --debug --startupbuffer 2 --bans 454,someplayer,489,512 --noclock --unrankedonly --maxconnectedgames 10 --maxconnectedgamesperuser 1 --minrank 5d --noautohandicapranked --maxhandicap 0 --fakerank 8d --boardsizes 19 --komis 7.5 --speeds live --maxmaintimelive 600 --maxperiodsliveranked 5 --maxperiodsliveunranked 10 --maxperiodtimeliveranked 60 --maxperiodtimeliveunranked 180 -- /home/amd2019/PhoenixGo/bazel-bin/mcts/mcts_main --gtp --config_path=/home/amd2019/PhoenixGo/etc/mcts_1gpu_beta.conf --v=1
-```
+Before you can play on official [OGS server](https://online-go.com/) server,
+ OGS admins will often ask that you run tests to make sure everything is running fine.
 
-### Submit Move Button
+You need to do these tests in [beta OGS server](https://beta.online-go.com/),
+ by adding the option [--beta](/docs/OPTIONS-LIST.md#beta).
 
-To avoid accidental misclicks while spectating a game from
-your bot account,
+When all testing is OK, you can remove the `--beta` and play in the real OGS.
 
-see: [Submit-Move Button](https://github.com/wonderingabout/gtp2ogs-tutorial#important-submit-move-button)
+## Submit Move Button
 
-### Show winrate and variations ingame
+To avoid accidental misclicks while spectating a game from your bot account, see
+ [Submit-Move Button](https://github.com/wonderingabout/gtp2ogs-tutorial#important-submit-move-button).
 
-**gtp2ogs has default support for showing ingame winrate and variations for some AI.**
+## Show winrate and variations ingame
 
-see: [--ogspv](/docs/OPTIONS-LIST.md/#ogspv)
+gtp2ogs has default support for showing ingame winrate and variations for some AI,
+ see: [--ogspv](/docs/OPTIONS-LIST.md/#ogspv)
 
 ## Community Involvment
```

Comment:
Involvement

Reply:
unrelated but it's actually not a default (not auto enabled) but a "native" feature

fixed typo too